Makefile

```diff
@@ -0,0 +1,77 @@
+.PHONY: help
+help:
```

Comment:
Please for this help task follow how was done at https://github.com/ioet/infra-terraform-modules/blob/fb270d25da76abacb4ce6fdac8ac3e3e8ef05a57/Makefile#L11

Reply:
Done